### PR TITLE
Update DCP_change_log.md

### PR DESCRIPTION
**Pull request by bentvandenheede**

### CODE REVIEW

File: release-notes/Cloud/DCP_change_log.md [hunk 1]
```diff
@@ -9,6 +9,18 @@ The dataminer.services platform gets updated continuously. This change log can h
 > [!NOTE]
 > Many features on dataminer.services are dependent on DxMs. You can find the change logs for these under [DxM release notes](xref:DxM_RNs_index).
 
+#### 13 August 2024 - Enhancement - Admin - Adding new user now blocked when user does not comply with configured allowed email domains [ID_40459]
+
+When you try to add a user to your organization in the Admin app, this will now only be possible if the user you add complies with the configured allowed email domains setting.
+
+#### 13 August 2024 - Enhancement - Admin - Non-compliant users in organization now shown [ID_40469]
```
[the remark this answers is not in the document]
Comment (bentvandenheede) [reply release-notes/Cloud/DCP_change_log.md:16]: Yes. The feature applies to the users overview of your organization.
The suggestion looks good.

File: release-notes/Cloud/DCP_change_log.md [hunk 2]
```diff
@@ -9,6 +9,18 @@ The dataminer.services platform gets updated continuously. This change log can h
...
+
+Users in an organization that do not comply with the configured allowed email domains setting will now be shown with a warning next to them, so that it is clear that these users were added before the allowed email domains setting was enforced.
+
+#### 13 August 2024 - Fix - Catalog - Deploy issues coordination name [ID_40470]
+
+When opening the deploy issues pop-up window. A user will now correctly see the name of the DMS which has an issue. Instead of a fixed text.
```
[the remark this answers is not in the document]
Comment (bentvandenheede) [reply release-notes/Cloud/DCP_change_log.md:22]: It is a bit difficult to describe in the change log.
I reffer to this fixed text.
image
The title of the pop-up window would show 'Deployment issues on ${coordinationName}' always.
With the fix, the title will now show the DMS name correctly (/dynamically). For example: my DMS is 'ITDMA', then my title would be 'Deployment issues on ITDMA'.

[the remark this answers is not in the document]
Comment (MariekeGO) [reply release-notes/Cloud/DCP_change_log.md:22]: In that case, I'd suggest we simplify the description to this:
```suggestion
When you opened the deployment issues pop-up window, this did not correctly display the name of the DMS with the issue. Now the DMS name will be shown correctly.
```

File: release-notes/Cloud/DCP_change_log.md [hunk 3]
```diff
@@ -9,6 +9,18 @@ The dataminer.services platform gets updated continuously. This change log can h
...
 
+#### 13 August 2024 - Enhancement - Admin - Adding new user now blocked when user does not comply with configured allowed email domains [ID_40459]
+
+When you try to add a user to your organization in the Admin app, this will now only be possible if the user you add complies with the configured allowed email domains setting.
+
+#### 13 August 2024 - Enhancement - Admin - Users overview now shows when users are non-compliant [ID_40469]
+
+Users in an organization that do not comply with the configured allowed email domains setting will now be shown with a warning next to them, so that it is clear that these users were added before the allowed email domains setting was enforced.
+
+#### 13 August 2024 - Fix - Catalog - Deploy issues coordination name [ID_40470]
```
Comment (MariekeGO) [inline release-notes/Cloud/DCP_change_log.md:20]:
```suggestion
#### 13 August 2024 - Fix - Catalog - Deployment issues pop-up window did not display DMS name correctly [ID_40470]
```

**Review by MariekeGO — APPROVED** [review]

**Pull request merged**